main/transaction/index.ts

```diff
@@ -173,4 +179,16 @@ async function sign(rawTx: TransactionData, signingFn: (tx: TypedTransaction) =>
   })
 }
 
-export { maxFee, populate, sign, signerCompatibility, londonToLegacy }
+function classifyTransaction(
```

Comment:
is it possible to just pass the transaction request object here as it has all the data we need right?